src/app/staticanalysis/modules/staticAnalysisCommon.js

```diff
@@ -0,0 +1,156 @@
+'use strict'
+
+var utils = require('../../utils')
+
+var nodeTypes = {
+  IDENTIFIER: 'Identifier',
+  MEMBERACCESS: 'MemberAccess',
+  FUNCTIONCALL: 'FunctionCall'
+}
+
+var basicTypes = {
+  UINT: 'uint256',
+  BOOL: 'bool',
+  ADDRESS: 'address',
+  BYTES32: 'bytes32'
+}
+
+var basicRegex = {
+  CONTRACTTYPE: '^contract ',
+  FUNCTIONTYPE: '^function \\('
+}
+
+var basicFunctionTypes = {
+  SEND: buildFunctionSignature([basicTypes.UINT], [basicTypes.BOOL], false),
+  CALL: buildFunctionSignature([], [basicTypes.BOOL], true),
+  DELEGATECALL: buildFunctionSignature([], [basicTypes.BOOL], false)
+}
+
+var lowLevelCallTypes = {
+  CALL: { ident: 'call', type: basicFunctionTypes.CALL },
+  CALLCODE: { ident: 'callcode', type: basicFunctionTypes.CALL },
+  DELEGATECALL: { ident: 'delegatecall', type: basicFunctionTypes.DELEGATECALL },
+  SEND: { ident: 'send', type: basicFunctionTypes.SEND }
+}
+
+var specialVariables = {
+  BLOCKTIMESTAMP: { obj: 'block', member: 'timestamp', type: basicTypes.UINT },
+  BLOCKHASH: {
+    obj: 'block',
+    member: 'blockhash',
+    type: buildFunctionSignature([basicTypes.UINT], [basicTypes.BYTES32], false)
+  }
```

Comment:
Yes would be the place to include that if needed in some kind of analysis. I just implemented what's needed right now. And there is more to come.